app/src/shared/components/DatePicker.jsx

```diff
@@ -1,61 +0,0 @@
-// @flow
-
-import React from 'react'
-import { I18n } from 'react-redux-i18n'
-
-import { isMobile as checkMobile } from '$shared/utils/platform'
-import WithCalendar, { type WithCalendarProps } from './WithCalendar'
-import CalendarIcon from './CalendarIcon'
-import Text from '$ui/Text'
-import dateFormatter from '$utils/dateFormatter'
-
-type Props = WithCalendarProps & {
-    format?: string,
-    onChange?: (Date) => void,
-    value?: Date | string,
-}
-
-const isMobile = checkMobile()
-const ISO_DATE_FORMAT = 'YYYY-MM-DD'
-
-class DatePicker extends React.Component<Props> {
```

Comment:
We could maybe leave this component for possible future use. Or maybe we can salvage the code from this PR if we ever need it 😄

Reply:
Yeah, I thought about it too, might be useful but better to remove so we don't get warnings about unused code. It will be there in the history if we ever need it.